src/shared/components/adminTabs/CurrentGame.tsx

```diff
@@ -110,7 +124,9 @@ const CurrentGame = (props: any) => {
 
   const handleRevealAnswer = (event: any, index: number) => {
     const revealedAnswers = game.revealedAnswers;
+   
     if (event.target.checked) {
+      playSound("src\\static\\correct_answer.mp3", 4000)
```

Comment:
This way of playing sounds may cause errors on another OS ( Mac/Linux ). 
try to import 
Here is an example `import correctAnswerSound from "../../static/correct_answer.mp3";`
and use it as such:
`playSound(correctAnswerSound, 4000)`